packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -28,6 +28,10 @@ typedef DiagnosticPropertiesTransformer = Iterable<DiagnosticsNode> Function(Ite
 /// and other callbacks that collect information describing an error.
 typedef InformationCollector = Iterable<DiagnosticsNode> Function();
 
+/// Signature for a function that demangles [StackTrace] objects into a format
+/// that can be parsed by [StackFrame].
```

Comment:
nit: might be worth mentioning that this should be used if someone is using `package:stack_trace`, with a small example of how.